test/integration/apiserver/apply/apply_test.go

```diff
@@ -3423,12 +3423,6 @@ func TestSubresourceField(t *testing.T) {
 		string(managedFields[1].FieldsV1.Raw) != `{"f:spec":{"f:replicas":{}}}` {
 		t.Fatalf(`Unexpected entry, got: %v`, managedFields[1])
 	}
-	if managedFields[2].Manager != "manager" ||
-		managedFields[2].Operation != "Update" ||
-		managedFields[2].Subresource != "status" ||
-		string(managedFields[2].FieldsV1.Raw) != `{"f:status":{"f:unavailableReplicas":{}}}` {
-		t.Fatalf(`Unexpected entry, got: %v`, managedFields[1])
-	}
```

Comment:
What was the reason to remove this?

Reply:
Oh I see I have forgotten about adding a call to scale as well, I think it would be nice to keep both